spec/computer_spec.rb

```diff
@@ -1,8 +1,32 @@
 require "computer"
+require "spec_helper"
+require "board"
 
 RSpec.describe Computer do
-  it "creates a new computer with a mark" do
-    computer = Computer.new("O")
+  board = Board.new(empty_board)
+  computer = Computer.new("O", board)
+
+  it "creates a new computer player with a mark" do
     expect(computer.mark).to eq("O")
   end
+  
+  it "selects an available move from the board" do
+    expect(computer.get_move.nil?).to eq(false)
+  end
+
+  it "is able to work out it's opponent's mark" do
+    expect(computer.opponent_mark).to eq("X")
+  end
+  
+  it "blocks it's opponent from winning the game" do 
+    board = Board.new(x_wins_on_next_move)
+    computer = Computer.new("O", board)
+    expect(computer.get_move).to eq("5")
```

Comment:
Trade-off here is that I don't know why it should be 5. It surprises me that the computer returns a string and not an integer. I have a hunch that's a detail of your human that has leaked to your computer.

Reply:
Resolved #40